ibm_catalog.json

```diff
@@ -0,0 +1,110 @@
+{
+  "products": [
+    {
+      "name": "terraform-ibm-cloudant",
+      "label": "IBM Cloudant module",
+      "product_kind": "solution",
+      "tags": [
+        "ibm_created",
+        "target_terraform",
+        "terraform",
+        "reference_architecture",
+        "solution"
+      ],
+      "keywords": [
+        "cloudant",
+        "IaC",
+        "infrastructure as code",
+        "terraform",
+        "solution",
+        "cloudant dedicated",
+        "cloud dedicated"
+      ],
+      "short_description": "Creates an IBM Cloudant Dedicated instance in IBM Cloud Dedicated environment.",
```

Comment:
Its possible we may have other flavours of the offer that do not require dedicated. So this description needs to higher level. Maybe go with: "Creates and configures an IBM Cloudant instance"